onnxruntime/test/contrib_ops/snpe_op_test.cc

```diff
@@ -152,8 +153,9 @@ TEST(Snpe_ConvertFromConcat, MimoSupport) {
 
   test.AddInput<float>("value0", {2, 2}, {11.0f, 12.0f, 13.0f, 14.0f});
   test.AddInput<float>("value1", {2, 2}, {21.0f, 22.0f, 23.0f, 24.0f});
-  test.AddOutput<float>("output", {4, 2}, {11.0f, 12.0f, 13.0f, 14.0f,
-                                           21.0f, 22.0f, 23.0f, 24.0f});
+  test.AddOutput<float>("output", {4, 2},
+                        {11.0f, 12.0f, 13.0f, 14.0f,
+                         21.0f, 22.0f, 23.0f, 24.0f});
   test.Run();
```

Comment:
clang-format doesn't join this into a single line if the opening '{' is on a new line.